src/constraints/electric_utility_constraints.jl

```diff
@@ -15,31 +15,40 @@ function add_export_constraints(m, p; _n="")
     WHL_benefit = 0
     NEM_techs = String[t for t in p.techs.elec if :NEM in p.export_bins_by_tech[t]]
     WHL_techs = String[t for t in p.techs.elec if :WHL in p.export_bins_by_tech[t]]
+    NEM_storage = String[b for b in p.s.storage.types.elec if :NEM in p.export_bins_by_storage[b]]
```

Comment:
This isn't really about what you added, but isn't NEM_techs just equal to techs_by_exportbins[:NEM]? Idk why we are recreating it.

Reply:
Yes I just double checked and I agree. We could probably replace all instances of NEM_techs and WHL_techs with their techs_by_export bin equivalent